### PR TITLE
Update banner

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

Updated the banner to promote the [governance page](https://govern.iota.org).  

govern_banner

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -26,11 +26,11 @@ module.exports = {
   themeConfig: {
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     announcementBar: {
-      id: 'assembly',
+      id: 'govern',
       content:
-        'Would you like to stake your tokens? <a target="_blank" href="https://assembly.sc">Read more</a> about Assembly, the upcoming permissionless Smart Contract network.',
-      backgroundColor: '#da7ad7',
-      textColor: '#000000',
+        'If you would like to get more involved in the future Governance of Shimmer, IOTA, and the Assembly network, join the discussions with the community in our <a target="_blank" href="https://govern.iota.org">governance forum 🏛️</a>',
```
Comment (Critical94) [inline docusaurus.config.js:31]:
```suggestion
        'If you would like to get more involved in the future governance of Shimmer, IOTA, and the Assembly network, join the discussions with the community in our <a target="_blank" href="https://govern.iota.org">governance forum</a> 🏛️',
```

**Review by Critical94 — CHANGES_REQUESTED** [review]

File: docusaurus.config.js (hunk 1 above)
Comment (Critical94) [reply docusaurus.config.js:31]: Not underlined icon is just my preference.
Nice color btw!

Comment (jlvandenhout) [reply docusaurus.config.js:31]: I support this message

**Review by jlvandenhout — APPROVED** [review]
With the changes proposed by Critical

**Review by Critical94 — APPROVED** [review]

**Pull request merged**